logger.go

```diff
@@ -29,6 +29,11 @@ type Logger struct {
 	// Flag for whether to log caller info (off by default)
 	ReportCaller bool
 
+	// FireAllHooks indicates that all hooks should be fired, even in case of an
+	// error. When off (default), when an error occurs firing one of the hooks,
```

Comment:
Maybe?
```
When false (default), the first hook that returns an error will stop any unfired hooks from firing and that error will be returned. When true, all hooks are fired and any errors are collected and returned.
```

Does this mean that mutliErr should be mentioned and exported?

i.e. I'm asking myself how would I process / handle the errors in the case of true and the documentation doesn't really give me any guidance.